source/_integrations/event.markdown

```diff
@@ -20,6 +20,8 @@ related:
 
 Events are signals that are emitted when something happens, for example, when a user presses a physical button like a doorbell or when a button on a remote control is pressed.
 
+The **Event** {% term integration %} provides {% term entities %} that trigger state change events, as do all other entity integrations.
```

Comment:
Is it worthwhile to introduce a term for the events on the event bus so we can easily refer to those?

Reply:
There is one <https://www.home-assistant.io/docs/glossary/#event>, it is just a bit brief. I know it needs to be updated, but I think it's a topic for another PR.